login-workflow/src/screens/Login.tsx

```diff
@@ -288,7 +333,7 @@ export const Login: React.FC = () => {
 
     return (
         <BrandedCardContainer loading={transitState.transitInProgress}>
-            {errorDialog}
+            {(loginErrorDisplayConfig.mode === 'dialog' || loginErrorDisplayConfig.mode === 'both') && errorDialog}
```

Comment:
Not sure if this is out of scope (it might be a not-insignificant lift), but would be nice if the popup also does not display if the empty string is passed as the error message.